packages/fx-core/src/plugins/resource/appstudio/index.ts

```diff
@@ -77,23 +77,17 @@ export class AppStudioPlugin implements Plugin {
     return ok(appStudioQuestions);
   }
 
-  public async updateApp(
-    appDefinition: IAppDefinition,
+  public async getAppDefinitionAndUpdate(
+    ctx: PluginContext,
     type: "localDebug" | "remote",
-    createIfNotExist: boolean,
-    teamsAppId?: string,
-    appStudioToken?: string,
-    logProvider?: LogProvider,
-    projectRoot?: string
+    manifest: TeamsAppManifest,
+    appStudioToken?: string
```

Comment:
appStudioToken can also be removed from parameter, you can get this token from context.

Reply:
Thanks! I have removed the appStudioToken from the parameter and get the token from the plugin Context.